content/docs/tutorials/streetlights.md

```diff
@@ -118,75 +118,106 @@ In this example, `light/measured` is the channel name your API will allow you to
             description: Date and time when the message was sent.
 ```
 
-It defines the content of the event using AsyncAPI schemas. It means that our event payload should contain an `id` and a `lumens` property —which are integers bigger than zero—, and a `sentAt` property that should be a string containing a date and time.
+`Payload` property defines the content of the event using AsyncAPI schemas. It means that your event payload should contain an `id` and a `lumens` property —which are integers bigger than zero—, and a `sentAt` property that should be a string containing a date and time.
 
 > JSON Schema Draft 07 is 100% compatible with AsyncAPI schemas.
 
-Cool! So we're done with our AsyncAPI file! Let's get into generating code.
+Cool! So you're done with your AsyncAPI file! Let's get into generating code.
 
 # Generating code
 
-To generate our code we'll use the [AsyncAPI Generator](https://github.com/asyncapi/generator) Node.js template.
-
-```bash
-npm install -g asyncapi-generator
-```
-
-(You might need to use sudo)
-
-Create a directory for your projects and step into it:
-
-```bash
-mkdir streetlights && cd "$_"
-```
-
-Create a file with the AsyncAPI machine-readable description we created before:
-
-```bash
-touch asyncapi.yaml
-# Open asyncapi.yaml and paste the definition
-```
-
-And now let's generate the code for it:
-```bash
-ag asyncapi.yaml nodejs -p server=mosquitto
-```
-
-And voilà!
+To generate your code you'll use the [AsyncAPI Generator](https://github.com/asyncapi/generator) Node.js template.
+
+1. Install the generator to use is at a command-line tool
+    ```bash
+    npm install -g asyncapi-generator
+    ```
+
+2. Create a directory for your projects and enter it:
+    ```bash
+    mkdir streetlights && cd "$_"
+    ```
+
+3. Create a file with the AsyncAPI machine-readable description you defined before:
+    ```bash
+    cat <<EOT >> asyncapi.yaml
```

Comment:
Watch out. `cat` doesn't work on Windows.

Reply:
how about if I add a sentence/note that on windows `cat` should be replaced with `type`

Reply:
Ok 👌